packages/x-charts-pro/src/FunnelChart/funnelSlots.types.ts

```diff
@@ -10,14 +16,27 @@ export interface FunnelChartSlots
     ChartsLegendSlots,
     ChartsTooltipSlots,
     ChartsOverlaySlots,
-    ChartsAxisSlots {}
+    ChartsAxisSlots,
+    Partial<ChartsSlots> {
+  /**
+   * Custom component for the toolbar.
+   * @default ChartsToolbar
+   */
+  toolbar?: React.ElementType<ChartsToolbarProps>;
+}
```

Comment:
To avoid error when copy pasting those, I defined the previous one as follows

```js
export interface ChartsTooltipSlots {
  /**
   * Custom component for the tooltip popper.
   * @default ChartsTooltipRoot
   */
  tooltip?: React.ElementType<ChartsTooltipProps>;
}

export interface ChartsTooltipSlotProps {
  /**
   * Custom component for the tooltip popper.
   * @default ChartsTooltipRoot
   */
  tooltip?: Partial<ChartsTooltipProps>;
}
```

So this could be replaced by something like `ChartsToolbarSlots`/`ChartsToolbarSlotProps`